client/reader/following-stream/index.jsx

```diff
@@ -97,6 +106,38 @@ module.exports = React.createClass( {
 				this.showSelectedPost( { replaceHistory: true } );
 			}
 		}
+
```

Comment:
should we also check the updateCount state in componentDidMount, in case the stream renders with updates available? we don't currently automatically accept new posts when mounting the stream.